jenkinsfiles/UB20_Check.groovy

```diff
@@ -80,7 +80,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```

Comment:
@DoyleLi, are you ok with such variable renaming? I have found you're the author of this file.

Reply:
Hi, it looks good to me. Thanks for notifying.

Reply:
Updated